docker-compose.yml

```diff
@@ -14,40 +14,30 @@ services:
     environment:
     - POSTGRES_PASSWORD=developmentpassword
 
-  middleman:
-    build:
-      context: .
-      target: middleman
-      cache_from:
-        - ${dockerHubUsername:-dfedigital}/teacher-training-api-middleman:master
-    image: ${dockerHubUsername:-dfedigital}/teacher-training-api-middleman:${GIT_BRANCH:-latest}
-
   web:
     build:
       context: .
       cache_from:
-        - ${dockerHubUsername:-dfedigital}/teacher-training-api:${GIT_BRANCH:-latest}
+        - ${dockerHubUsername:-dfedigital}/teacher-training-api:${TAG:-latest}
         - ${dockerHubUsername:-dfedigital}/teacher-training-api-middleman:master
-    image: ${dockerHubUsername:-dfedigital}/teacher-training-api:${GIT_BRANCH:-latest}
+    image: ${dockerHubUsername:-dfedigital}/teacher-training-api:${TAG:-latest}
```

Comment:
call it IMAGE_TAG to keep it consistent with the Makefile?

Reply:
done, thanks